### DISCUSSION TITLE
Moongates work on double click

### DISCUSSION THREAD

**Discussion by DominusExult** [exult/exult#473] [Announcements]
In Exult you can double click a moongate and if it is in reach it teleports you to its destination. The original does not do that, you have to walk through the gate.
No idea if this should be considered a bug or a feature :)

Example moongate on the isle of fire: 08af, 0626

**Comment by jharrison03** [comment]
> Example moongate on the isle of fire: 08af, 0626

If this is consistent behavior, I would think this is an improvement, if a minor one.
